content/faq.md

```diff
@@ -0,0 +1,157 @@
+
+#### Isn’t Proof of Work the best validation mechanism?
+
+Proof of Work (PoW) was the first mechanism to solve the double spend problem without requiring a trusted 3rd party. [The XRP Ledger’s consensus mechanism](intro-to-consensus.html) solves the same problem in a far faster, cheaper and more energy efficient way.
+
+#### How can a blockchain be sustainable?
+
+It’s been widely reported that Bitcoin’s energy consumption, as of 2021, is equivalent to that used by Argentina, with much of the electricity Bitcoin miners use coming from polluting sources. The XRP Ledger confirms transactions through a “[consensus](intro-to-consensus.html)” mechanism - which does not waste energy like proof of work does - and leverages carbon offsets to be [one of the first truly carbon neutral blockchains](https://ripple.com/ripple-press/ripple-leads-sustainability-agenda-to-achieve-carbon-neutrality-by-2030/). Explore the energy consumption of XRP compared to cash, credit cards and other popular cryptocurrencies with the [Green Currency Calculator](carbon-calculator.html).
+
+#### Can currencies other than XRP be traded through XRPL?
+
+Yes, the XRP Ledger was built specifically to be able to tokenize arbitrary assets, such as USD, EUR, Oil, Gold, reward points, and more. Any currency can be issued on the XRP Ledger. This is illustrated by XRPL’s growing community that backs a variety of fiat and crypto tokens.
+
+#### Isn’t XRPL only for payments?
+
+Although XRPL was initially developed for payment use cases, both the ledger and its native digital asset XRP are increasingly popular for a range of innovative blockchain use cases. New standards proposals for issuing NFTs, XRPL Labs’ hooks amendment for smart contract functionality, and federated sidechains are all currently works in progress.
+
+
+## Validators and Unique Node Lists
+
+#### What service do transaction validators provide?
+
+Validators determine if transactions meet protocol requirements, and are therefore “valid.” The service validators uniquely provide is administratively grouping transactions into ordered units, agreeing on one such ordering specifically to prevent double spending. <!-- STYLE_OVERRIDE: therefore -->
+
+See [Consensus](consensus.html) and the [Ripple Labs Tech Talk: Understanding Consensus](https://ripple.com/insights/ripple-labs-tech-talk-consensus-within-the-ripple-protocol/) for more information about the consensus process.
+
+
+#### How much does it cost to run a validator?
+
+Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of its electricity use.
```

Comment:
```suggestion
Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of electrical usage.
```

Reply:
I prefer the existing wording here. (Fun fact: searching "electrical usage" turns up a lot of results for "electricity use" instead.)